e107_handlers/user_model.php

```diff
@@ -1285,10 +1297,13 @@ public function renderEmail($type, $userInfo)
 			$EMAIL_TEMPLATE['signup']['bcc']			= $SIGNUPEMAIL_BCC;
 			$EMAIL_TEMPLATE['signup']['attachments']	= $SIGNUPEMAIL_ATTACHMENTS;		
 			$EMAIL_TEMPLATE['signup']['body']			= $SIGNUPEMAIL_TEMPLATE;
-			
-			$EMAIL_TEMPLATE['quickadduser']['body']		= $QUICKADDUSER_TEMPLATE['email_body'];
-			$EMAIL_TEMPLATE['notify']['body']			= $NOTIFY_TEMPLATE['email_body'];
-			
+
+			// Both template vars are not defined...
+			//$EMAIL_TEMPLATE['quickadduser']['body']		= $QUICKADDUSER_TEMPLATE['email_body'];
+			//$EMAIL_TEMPLATE['notify']['body']			= $NOTIFY_TEMPLATE['email_body'];
+			$EMAIL_TEMPLATE['quickadduser']['body']	= '';
+			$EMAIL_TEMPLATE['notify']['body']			= '';
+

```

Comment:
Thank you!. Could you explain the thinking behind setting `$EMAIL_TEMPLATE['quickadduser']['body'] ` etc. to empty values?

Reply:
This was just a quick fix for the missing variables.
I did a search on the whole codebase and i couldn't find a var called `$QUICKADDUSER_TEMPLATE` or `$NOTIFY_TEMPLATE`. So instead making both unknown vars global to be able to use them in the method, i decided to just remove them by commenting out and setting an empty string instead.

Do you know, where to find those vars or what content to use instead for the `$EMAIL_TEMPLATE` var?

Reply:
I think it's a BC fix. We should probably check if `$QUICKADDUSER_TEMPLATE['email_body']` is not empty, and assign it to `$EMAIL_TEMPLATE['quickadduser']['body']` if it has a value. Similar with 'notify'

Reply:
I will make that missing vars global and use vartrue() for the assignment.
But first I must sort out my line endings issue... Don't want to screw up another file...

Reply:
@CaMer0n Pushed a new commit with the proposed changes to the 2 vars.